### PR TITLE
[ExportVerilog] Add option to not emit location info

### PR DESCRIPTION
**Pull request by rsetaluri**

### CODE REVIEW

File: lib/Conversion/ExportVerilog/ExportVerilog.cpp [hunk 1]
```diff
@@ -507,6 +507,8 @@ getLocationInfoAsString(const SmallPtrSet<Operation *, 8> &ops,
     return str;
   case LoweringOptions::LocationInfoStyle::WrapInAtSquareBracket:
     return "@[" + str + ']';
+  case LoweringOptions::LocationInfoStyle::None:
+    return "";
```
Comment (uenoku) [inline lib/Conversion/ExportVerilog/ExportVerilog.cpp:511]: Could you return early without constructing `str`? i.e.
```cpp
if(style == LoweringOptions::LocationInfoStyle::None)
    return "";
auto str = getLocationInfoAsStringImpl(ops);
...
```

**Review by uenoku — APPROVED** [review]
Thanks! I have one nit comment, otherwise LGTM!

File: lib/Conversion/ExportVerilog/ExportVerilog.cpp (hunk 1 above)
Comment (rsetaluri) [reply lib/Conversion/ExportVerilog/ExportVerilog.cpp]: done!

**Comment by rsetaluri** [comment]
> Thanks! I have one nit comment, otherwise LGTM!

Do I need approval for the workflows to be run so it can be merged? Thanks.

**Comment by uenoku** [comment]
I'd like to merge this. Thank you for working on this!

**Pull request merged**